workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -138,13 +138,38 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
         if (isGroupNotFoundError(error)) {
           throw Boom.notFound(error.message);
         }
+        if (isTooManyRequestsError(error)) {
```

Comment:
the status you audit log in the service side should match what you throw here or be more specific (not less specific)